### PR TITLE
Added text files for kml, mscolab, wms, and views tutorials.

### PR DESCRIPTION
**Pull request by risehr**
Added text files for kml, mscolab, wms and views tutorials
Updated the tutorials.rst file

### CODE REVIEW

File: tutorials/textfiles/tutorial_mscolab.txt [hunk 1]
```diff
@@ -0,0 +1,199 @@
+mission support collaboration
+
+MS Colab stores data in an online server, and can be used to access the data remotely as also working in a team where everyone contributes his part. It is used for collaborating with the users as a team together and working on a shared MScolab project.
```
Comment (ReimarBauer) [inline tutorials/textfiles/tutorial_mscolab.txt]: no blank
MSColab

File: tutorials/textfiles/tutorial_mscolab.txt [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+Filling new user Details, and clicking OK.
+
+As, the user exists, we go back to cancel and enter the login page again!
+
+logging into my profile by entering my user name and password.
+
+Then clicking on the login button.
+
+Now the user John Doe is logged in to MScolab.
```
Comment (ReimarBauer) [inline tutorials/textfiles/tutorial_mscolab.txt:24]: MSColab

review for same writing

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please check wording of MSColab

File: tutorials/textfiles/tutorial_mscolab.txt (hunk 1 above)
Comment (risehr) [reply tutorials/textfiles/tutorial_mscolab.txt]: Okay, I will change that.

File: tutorials/textfiles/tutorial_mscolab.txt (hunk 2 above)
Comment (risehr) [reply tutorials/textfiles/tutorial_mscolab.txt:24]: Sure, doing soon.

**Comment by risehr** [comment]
Rectified the spellings. Done!

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**